aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -361,7 +361,7 @@ module AptosFramework::Stake {
     }
 
     #[test(core_resources = @CoreResources, account_1 = @0x123, account_2 = @0x234, account_3 = @0x345)]
-    fun test_basic_delegation(
+    public(script) fun test_basic_delegation(
```

Comment:
yeah

Reply:
security!

there's an implicit notion that (script) will have lesser security guarantees since the caller can create things outside the context of the VM. In practice, however, I can't imagine such an attack as we don't expose richness to the function arguments.